src/Extensions/MemberExtension.php

```diff
@@ -68,4 +64,17 @@ public function getJWTData()
 
         return Convert::raw2json($data);
```

Comment:
raw2json has been superseeded with direct json_encode, without calling `Convert` class (unrelated but related, might as well fix it up straight away)